operators/src/engine/result_descriptor.rs

```diff
@@ -69,10 +71,25 @@ impl ResultDescriptor for RasterResultDescriptor {
 }
 
 /// A `ResultDescriptor` for vector queries
-#[derive(Debug, Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Serialize, Deserialize)]
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
 pub struct VectorResultDescriptor {
     pub data_type: VectorDataType,
     pub spatial_reference: SpatialReferenceOption,
+    pub columns: HashMap<String, FeatureDataType>,
+}
+
+impl VectorResultDescriptor {
+    /// Create a new `VectorResultDescriptor` by only using new `columns`
```

Comment:
why is columns in tick when there is no parameter named columns? Also why do they all have to be new? Then I wouldn't use map but new

Reply:
ba5480b 

You don't have to, you only get a reference of the hash map.